src/algebra/divisibility.lean

```diff
@@ -89,13 +91,11 @@ section monoid
 
 variables [monoid α]
 
-@[refl, simp] theorem dvd_refl (a : α) : a ∣ a :=
-dvd.intro 1 (mul_one _)
-
-lemma dvd_rfl {a : α} : a ∣ a :=
-dvd_refl a
```

Comment:
Personally I think this way around makes more sense, as the reading is:
* `dvd_refl` is the standard way to state a lemma with the usual argument explicitness
* `dvd_rfl` is a shorthand that goes against the rules